src/Sulu/Component/PHPCR/PathCleanup.php

```diff
@@ -65,7 +65,7 @@ class PathCleanup implements PathCleanupInterface
      */
     public function cleanup($dirty, $languageCode)
     {
-        $clean = strtolower($dirty);
+        $clean = mb_strtolower($dirty);
```

Comment:
I don't know, the manual also says that this is a very slow function... Maybe we should try to get the locale settings right on our machines?